package/usr/local/bin/hassbian-config

```diff
@@ -135,65 +133,47 @@ function share-log {
   return 0
 }
 
-function install-suite {
-  if [ "$(id -u)" != "0" ]; then
-    echo "This script must be run with sudo. Use 'sudo hassbian-config install $1'" 1>&2
-    return 1
-  fi
-  if [ "$DEBUG" == "true" ]; then
-    set -x
-  fi
-  if [ ! -f $SUITE_CONTROL_DIR/"$1" ]; then
-    touch $SUITE_CONTROL_DIR/"$1"
-    echo "SCRIPTSTATE=uninstalled" > $SUITE_CONTROL_DIR/"$1"
-  fi
-    SUITESTATE=$(grep "SCRIPTSTATE" $SUITE_CONTROL_DIR/"$1" | awk -F'=' '{print $2}')
-    source $SUITE_INSTALL_DIR/"$1".sh
-  if [ "$FORCE" == "true" ]; then
-    "$1"-install-package | tee $LOGFILE
-    sed -i -- 's/SCRIPTSTATE='"$SUITESTATE"'/SCRIPTSTATE=installed/g' $SUITE_CONTROL_DIR/"$1"
-  elif [ "$SUITESTATE" == "installed" ]; then
-    echo "$1 is already installed, trying upgrade script..."
-    UPGRADE=$(grep "$1"-upgrade-package $SUITE_INSTALL_DIR/"$1".sh)
-  if [ "$UPGRADE" == "" ]; then
-    echo "Upgrade script is not available..."
-    echo "You can force run the install script like this:"
-    echo "sudo hassbian-config -f install $1"
-    return 0
-  fi
-    "$1"-upgrade-package | tee $LOGFILE
-    sed -i -- 's/SCRIPTSTATE='"$SUITESTATE"'/SCRIPTSTATE=installed/g' $SUITE_CONTROL_DIR/"$1"
+function run-suite { #This is the function the actually run install/upgrade.
+  source "$SUITE_INSTALL_DIR/$2".sh #Sourcing the script file.
+  "$2"-show-short-info && "$2"-show-copyright-info && printf "\\n" #Prints suite-show-short-info & suite-show-copyright-info to console.
+  if [ "$DEBUG" == "true" ]; then set -x; fi #Activating debug if --debug is used.
+  "$2"-"$1"-package #Running the script function install/upgrade SUITE.
+  RETURN=("$?") # Return value after script execution.
+  if [ "$DEBUG" == "true" ]; then set +x; fi #Deactivating debug if --debug is used.
+  if [ "$RETURN" == "0" ]; then STATE="installed"; else printf "\\e[0mIf you have issues with this script, please say something in the #devs_hassbian channel on Discord.\\n" && STATE="failed"; fi #Set suite state to installed if 0 is returned, failed otherwise.
+  echo "SCRIPTSTATE=$STATE" > "$SUITE_CONTROL_DIR/$2" #Setting status in control file.
+  return 0
+}
+
+function install-suite { #This function do checks if we want/can to install.
```

Comment:
Wording in explanation is a bit of.
Change to 
`#This function do checks if we can/want to install.`

Reply:
I read this 5 times before seeing the change...